### DISCUSSION TITLE
Disable Collider

### DISCUSSION THREAD

**Discussion by Wuketuke** [dimforge/rapier#755] [Q&A]
I need a way to disable all collisions of a certain collider, but I could not find any way of achieving that. I would have guessed that setting the `Visibility` component to `Hidden` would do the trick, but that does nothing.

**Comment by nstoddard** [comment]
One way would be to use `collider.set_collision_groups(InteractionGroups::none())` (I haven't tested this, but it should work). Or if you're using bevy, I think adding the component `CollisionGroups { memberships: Group::NONE, filters: Group::NONE }` should work.

**Comment by ThierryBerger** [comment]
There is [ColliderDisabled](https://docs.rs/bevy_rapier3d/latest/bevy_rapier3d/geometry/struct.ColliderDisabled.html) too.
